src/main/java/com/veryfi/VeryfiClient.java

```diff
@@ -112,26 +112,27 @@ public String getUrl() {
 
     /**
      * Submit the HTTP request.
-     * @param httpVerb HTTP Method
+     * @param httpMethod HTTP Method
      * @param endpointName Endpoint name such as 'documents', 'users', etc.
      * @param requestArguments JSON payload to send to Veryfi
      * @return A JSON String of the response data.
      * @throws VeryfiClientException if there is any error in making request to veryfi APIs
      */
-    public String request(String httpVerb, String endpointName, Map<String, Object> requestArguments) throws VeryfiClientException {
-        return request(httpVerb, endpointName, requestArguments, false);
+    public String processRequest(String httpMethod, String endpointName, Map<String, Object> requestArguments) throws VeryfiClientException {
+        return processRequest(httpMethod, endpointName, requestArguments, false);
     }
 
     /**
      * Submit the HTTP request.
-     * @param httpVerb HTTP Method
+     * @param httpMethod HTTP Method
      * @param endpointName Endpoint name such as 'documents', 'users', etc.
      * @param requestArguments JSON payload to send to Veryfi
      * @param isFileStream A boolean to check if the file stream is available default is false
      * @return A JSON of the response data.
      * @throws VeryfiClientException if there is any error in making request to veryfi APIs
      */
-    public String request(String httpVerb, String endpointName, Map<String, Object> requestArguments, boolean isFileStream) throws VeryfiClientException {
+    public String processRequest(String httpMethod, String endpointName, Map<String, Object> requestArguments, boolean isFileStream) throws VeryfiClientException {
+        logger.info("Processing Request with httpMethod[{httpMethod}],endpointName[{}]-[s t a r t]",httpMethod,endpointName);
```

Comment:
`httpMethod[{httpMethod}]`  should be replaced by `httpMethod[{}]`
nit: spacing with commas